docs/quickget.1.md

```diff
@@ -22,28 +22,45 @@ configuration for `quickemu` to use to build and run
 
 # OPTIONS
 
```

Comment:
@flexiondotorg These changes to quickget options need reflecting in the wiki at [here](https://github.com/quickemu-project/quickemu/wiki/06-Advanced-quickget-features#command-line-arguments)
, which still has the old help output